app/models/user.rb

```diff
@@ -1,12 +1,20 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please add :confirmable method here, as per the requirement.

Reply:
Alright

Reply:
fixed